### PR TITLE
Modernize Python versions.

### PR DESCRIPTION
**Pull request by icemac**
- drop support for Python 3.4.
- add support for Python 3.8.

@jugmac00 I saw this branch lying around and thought It might be a good candidate for a PR. 😀

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
+
 script:
   - coverage run -m zope.testrunner --test-path=src  --auto-color --auto-progress
 
 after_success:
   - coveralls
-notifications:
+
+  notifications:
   email: false
```
Comment (mgedmin) [inline .travis.yml]: This looks like an editing mistake that introduces a YAML syntax error.  Let's fix it.

File: CHANGES.rst [hunk 2]
```diff
@@ -5,7 +5,9 @@ CHANGES
 5.1.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Add support for Python 3.8.
+
+- Drop support for Python 3.4.
```
Comment (mgedmin) [inline CHANGES.rst]: Bump version to 5.2.0?

File: .travis.yml (hunk 1 above)
Comment (jugmac00) [reply .travis.yml]: Thank you for the hint.

File: CHANGES.rst (hunk 2 above)
Comment (jugmac00) [reply CHANGES.rst]: Thank you for the hint.

**Comment by jugmac00** [comment]
Hi @icemac - thank you for reminding me on this branch - which I have totally forgotten about.

When I remember correctly, I created quite a bunch of branches back on Oct 1st.

Before creating a pr I wanted to wait until the tests passed - but I obviously closed one browser tab too early :-)

@mgedmin Thank you very much for the review! And sorry for these careless mistakes.

**Pull request merged**